docs/en_US/Tutorial/Nnictl.md

```diff
@@ -704,6 +704,21 @@ Debug mode will disable version check function in Trialkeeper.
 ### Manage webui
 
 * __nnictl webui url__
+  * Description
+
+    Show an experiment's webui url
+
+  * Usage
+
+    ```bash
+    nnictl webui url [options]
```

Comment:
`options` -> `experiment_id`

Reply:
I think here we should follow the same format as above.

Reply:
i agree, that's why i suggest changing `options` to `experiment_id`...